src/main/java/novamachina/exnihilosequentia/common/tileentity/barrel/mode/FluidTransformBarrelMode.java

```diff
@@ -85,16 +86,17 @@ public CompoundNBT write() {
 
     @Override
     protected void spawnParticle(AbstractBarrelTile barrelTile) {
-        ((ServerWorld) barrelTile.getLevel())
-            .sendParticles(ParticleTypes.EFFECT,
-                barrelTile.getBlockPos().getX() + barrelTile.getLevel().random.nextDouble(),
-                barrelTile.getBlockPos().getY() + barrelTile.getLevel().random.nextDouble(),
-                barrelTile.getBlockPos().getZ() + barrelTile.getLevel().random.nextDouble(),
+        ServerWorld level = (ServerWorld) barrelTile.getLevel();
+        Preconditions.checkNotNull(level, "Level is null.");
+        level.sendParticles(ParticleTypes.ASH,
+                barrelTile.getBlockPos().getX() + (.2d + (.8d - .2d) * level.random.nextDouble()),
+                barrelTile.getBlockPos().getY() + 1.2d,
+                barrelTile.getBlockPos().getZ() + (.2d + (.8d - .2d) * level.random.nextDouble()),
                 1,
-                0.0,
-                0.0,
-                0.0,
-                0.05);
+                0,
+                0,
+                0,
+                0.01);
```

Comment:
I would like to keep this particle the same so that we can distinguish between processes.

Reply:
Done in 80f7db1

![image](https://user-images.githubusercontent.com/52864251/142794108-1d76fbe3-4038-4441-9d7d-6dcce06956fd.png)